### ISSUE TITLE
Add One-Time Password

### ISSUE THREAD

**Issue by agdamsbo** [jd1378/otphelper#53]
Hi. Thank you for this great tool. I saw you just added "One Time Password". I have a service sending passwords with the keyword "One-Time Password: P4SSW0RD". With the passworg being an alphanumeric, upper case string of length 8. Could that be added?

**Comment by jd1378** [comment]
Hi
Alphanumeric is supported
I just need to add One-time password now i guess
 Please copy the actual message here as well for testing

**Comment by agdamsbo** [comment]
Cool. This is an example of the full text:

```
Danmarks Statistik SMS Token

One-Time Password:
TNPVXNHD

Expiration Date: Fri Jan 05 12:58:48 CET 2024
```

**Comment by jd1378** [comment]
released in [v1.11.0](https://github.com/jd1378/otphelper/releases/tag/v1.11.0)
